system-int-test/test/cmr/system_int_test/search/service/collection_service_search_test.clj

```diff
@@ -47,11 +52,13 @@
     (is (= [200 coll-json]
            [status results]))))
 
-(defn- verify-collection-has-formats
-  "Verify the collection in ATOM and JSON response has-formats field has the given value"
-  [coll has-formats]
-  (verify-collection-atom-has-formats coll has-formats)
-  (verify-collection-json-has-formats coll has-formats))
+(defn- assert-collection-atom-json-result
+  "Verify collection in ATOM and JSON response has-formats, has-variables and associations fields"
+  ([coll has-formats serv-concept-ids]
+   (assert-collection-atom-json-result coll has-formats serv-concept-ids nil))
+  ([coll has-formats serv-concept-ids var-concept-ids]
+   (assert-collection-atom-result coll has-formats (boolean (seq var-concept-ids)))
```

Comment:
Not sure if `some?` or `boolean` makes more sense. I guess the only time it would behave different is if the value was literally `false`.

Reply:
`some?` may be more readable in this case. I will change it in my ongoing work and include it in my next PR.